### ISSUE TITLE
Add module: Editoria11y Accessibility Checker

### ISSUE THREAD

**Issue by trubach** [ATLAS-Illinois/Illinois-Framework-Theme#1025]
The [Editorial11y Drupal module](https://www.drupal.org/project/editoria11y) should be added to the Framework. It presents accessibility issues to the content editors in either the editing mode or on the rendered page.

A demo is available at https://editoria11y.princeton.edu/demo/

**Comment by trubach** [comment]
Changes to add Editoria11y are in PRs in core and profile at:

web-illinois/illinois_framework_core#133
web-illinois/illinois_framework_profile#52

**Comment by trubach** [comment]
TODO: Before closing issue, verify that the content manger role gets permissions to use Editoria11y through the drush config-distro-update command. If it doesn't apply, we should modify the core update hook so that it adds the permissions to the role through that. The added permissions are:

  - 'mark as hidden in editoria11y'
  - 'view editoria11y checker'

**Comment by trubach** [comment]
Confirmed permissions are set and documentation is available at https://drupal.webtheme.illinois.edu/tutorials/accessibility-web/how-use-editoria11y-built-accessibility-checker
